include/engine/api/route_api.hpp

```diff
@@ -68,6 +68,11 @@ class RouteAPI : public BaseAPI
         response.values["waypoints"] = BaseAPI::MakeWaypoints(all_start_end_points);
         response.values["routes"] = std::move(jsRoutes);
         response.values["code"] = "Ok";
+        auto data_timestamp = facade.GetTimestamp();
+        if (!data_timestamp.empty())
+        {
+            response.values["data_version"] = facade.GetTimestamp();
```

Comment:
This could just be `response.values["data_version"] = data_timestamp;`, right?

Reply:
Sure